### PR TITLE
Interact with dropdown, checkbox, alert

### LINKED ISSUE #6: Interact with dropdown, checkbox, alert
Go to [this site](https://the-internet.herokuapp.com/)
1. Select `Dropdown` and perform select option using selenium `Select`
2. Select `Checkboxes` and perform action check, uncheck
3. Select `JavaScript Alerts` and perform actions `acept` `dismiss` `getText` `sendKeys` to alert popup

### PR DESCRIPTION
**Pull request by NguyenLan94**

### CODE REVIEW

File: src/test/java/Homework/DropdownCheckboxAlertTest.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
+package Homework;
```
Comment (justthieenj) [inline src/test/java/Homework/DropdownCheckboxAlertTest.java]: move to package `theinternet`
also the package name should be in all lower cases

File: src/test/java/Homework/DropdownCheckboxAlertTest.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import org.testng.annotations.Test;
+
+public class DropdownCheckboxAlertTest extends BaseTest {
+
+    @Test
+    public void dropdownInteraction() {
+        dr.get("https://the-internet.herokuapp.com/");
+        dr.findElement(By.cssSelector("#content [href$='dropdown']")).click();
+        Select dropdown = new CustomSelect(dr.findElement(By.cssSelector("#dropdown")));
+        dropdown.selectByVisibleText("Option 2");
```
Comment (justthieenj) [inline src/test/java/Homework/DropdownCheckboxAlertTest.java]: use `Assert` to verify if the test is meet the expectation

File: src/test/java/Homework/DropdownCheckboxAlertTest.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        Utils.sleep(2);
```
Comment (justthieenj) [inline src/test/java/Homework/DropdownCheckboxAlertTest.java]: no need to add sleep here, it's for demo or debug only

File: src/test/java/Homework/DropdownCheckboxAlertTest.java (hunk 1 above)
Comment (NguyenLan94) [reply src/test/java/Homework/DropdownCheckboxAlertTest.java]: resolved

File: src/test/java/Homework/DropdownCheckboxAlertTest.java (hunk 2 above)
Comment (NguyenLan94) [reply src/test/java/Homework/DropdownCheckboxAlertTest.java]: resolved

File: src/test/java/Homework/DropdownCheckboxAlertTest.java (hunk 3 above)
Comment (NguyenLan94) [reply src/test/java/Homework/DropdownCheckboxAlertTest.java]: resolved

**Review by justthieenj — APPROVED** [review]

**Pull request merged**